kahuna/app/lib/KahunaConfig.scala

```diff
@@ -21,4 +23,22 @@ class KahunaConfig(resources: GridConfigResources) extends CommonConfig(resource
   val supportEmail: Option[String]= stringOpt("links.supportEmail").filterNot(_.isEmpty)
 
   val frameAncestors: Set[String] = getStringSet("security.frameAncestors")
+  val fileMetadataConfigs: List[FileMetadataConfig] = configObjectOpt("filemetadata.configurations") match {
+    case Some(config) => config.unwrapped()
+      .asInstanceOf[java.util.Map[String, java.util.HashMap[String, java.util.HashMap[String, Object]]]]
```

Comment:
Can we extract them without using `asInstanceOf`?

Reply:
No, from the structure of the HOCON config with dynamic paths. More than happy to get any pointers on how to achieve this without casting.

Reply:
The cast `.asInstanceOf[java.util.Map[String, java.util.HashMap[String, java.util.HashMap[String, Object]]]]` can be cleaner by casting to `ConfigObject` instead i.e. `asInstanceOf[ConfigObject]`